keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/Verification.kt

```diff
@@ -13,4 +14,9 @@ interface Verification {
    * in the database.
    */
   val id: String
+
+  /**
+   * Generate a URL that a user can be directed to in order to view the current state of a verification
+   */
+  fun link(state: VerificationState) : String? = null
```

Comment:
nit: should this be a `val` instead, since you're not naming it with a verb (`getLink`)?

Reply:
I ended up renaming this to `getLink` instead